src/object/srv_obj_migrate.c

```diff
@@ -1964,9 +1964,7 @@ migrate_system_enter(struct migrate_pool_tls *tls, int tgt_idx, bool *yielded)
 		D_DEBUG(DB_REBUILD, DF_RB ": tgt%d:%u max %u\n", DP_RB_MPT(tls), tgt_idx, tgt_cnt,
 			tls->mpt_inflight_max_ult / dss_tgt_nr);
 		*yielded = true;
-		ABT_mutex_lock(tls->mpt_inflight_mutex);
-		ABT_cond_wait(tls->mpt_inflight_cond, tls->mpt_inflight_mutex);
-		ABT_mutex_unlock(tls->mpt_inflight_mutex);
+		dss_sleep(1);
```

Comment:
just out of curiosity, do you have any sense  of whether adjusting this to other small delays > 1 millisecond has a significant effect on the rate of objects processed?